docs/watcher/run-watcher.md

```diff
@@ -0,0 +1,611 @@
+---
+id: run-watcher
+title: How to Run a Watcher
+sidebar_label: Run a Watcher
+---
+
+*By the end of this guide, you should know how to run a Watcher on your local machine, on VPS, or a bare-metal server. The guide is useful for enterprise clients who want to integrate with the OMG Network.*
+
+## Goals
+
+You should use this guide if you need to accomplish one of the following goals:
+- Rely on a personal Watcher to verify transactions and receive network's events
+- Host a redundant Watcher node to secure the network
+- Have an ability to challenge UTXOs
```

Comment:
is this correct? perhaps a more wholesome description should be used, for example, "be an active participant of the network"

Reply:
These are kind of valid reasons. I'll add yours too : )